install_config/topics/ocp_osp_sdn.adoc

```diff
@@ -0,0 +1,35 @@
+=== {product-title} SDN
+
+The default {product-title} SDN is
+xref:../architecture/networking/sdn.adoc#architecture-additional-concepts-sdn[OpenShiftSDN].
+There is another option: use xref:../install_config/configure_kuryrsdn.adoc#kuryr-sdn-and-openshift[Kuryr SDN].
+
+=== Kuryr SDN
+
+Kuryr is a CNI plugin using Neutron and Octavia to provide networking for pods
+and services. It is primarily designed for OpenShift clusters running on
+OpenStack virtual machines. Kuryr improves the network performance by plugin OCP
+pods into OpenStack SDN. In addition it provides interconnectivity between OCP
+pods and OSP virtual instances.
+
+Kuryr is recommended for {product-title} deployments:
```

Comment:
This sentence has some awkward constructions that won't translate well. I suggest changing it to something like this.

```
Kuryr is recommended for {product-title} deployments on 
encapsulated OpenStack tenant networks in order to avoid double
encapsulation, such as running an encapsulated OpenShift SDN over an encapsulated
OpenStack network. Kuryr is recommended whenever VXLAN, GRE, or GENEVE are required.
```